lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -1284,17 +1302,33 @@ def show_module_set(type, module_set, regex = nil, minrank = nil, opts = nil) #
             print(tbl.to_s)
           end
 
-          def generate_module_table(type, search_term = nil) # :nodoc:
-            Table.new(
-              Table::Style::Default,
-              'Header'     => type,
-              'Prefix'     => "\n",
-              'Postfix'    => "\n",
-              'Columns'    => [ '#', 'Name', 'Disclosure Date', 'Rank', 'Check', 'Description' ],
-              'SearchTerm' => search_term
-            )
+          def generate_module_table(type, search_terms = [], row_filter = nil) # :nodoc:
```

Comment:
does this have any existing tests you can add on to?

Reply:
We can add unit tests for the formatters/table instead of testing this directly IMO 🎉

Reply:
As above, what are your thoughts on renaming this back to:

```
```suggestion
          def generate_module_table(type, search_term = nil, highlight_terms =[]) # :nodoc:
```

The calling code will need to have its arguments switched round, and this method will need to be updated to handle that rename too :+1: